modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaGetClassNamesMethodCheck.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetClassNamesMethodCheck
+	extends BaseUpgradeMatcherReplacementCheck {
+
+	@Override
+	protected String afterFormat(
+		String fileName, String absolutePath, String content,
+		String newContent) {
+
+		if (fileName.endsWith(".java")) {
+			newContent = addNewImports(newContent);
+		}
+
+		return newContent;
+	}
```

Comment:
Hi @TamyrisTorres, you don't need to overwrite this method to add this check in checks that are only for java classes.

Reply:
Hi @nicolas-sss, I now need this method because I saw one case in the "jspf" file. So, I made other changes and included a new "jspf" file test. Please, Could you review it again?